src/Api/Managers/NotificationManager.cs

```diff
@@ -1,65 +1,87 @@
-﻿using BLL.Managers;
-using BOL.Data;
+﻿using BOL.Data;
 using BOL.Models;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace BLL.Managers
 {
     public class NotificationManager : BaseManager
     {
         public NotificationManager(CmsDbContext dbContext) : base(dbContext) { }
 
-        public virtual IEnumerable<Notification> CurrentUser(User userId)
+        public virtual IEnumerable<List<Notification>> CurrentUser(User userId)
         {
             return (from n in dbContext.Notifications
-                    where n.Id == userId.Id
-                    select n).ToList();
+                    join u in dbContext.Users
+                    on n.UserId equals userId.Id
+                    orderby n.TimeStamp descending
+                    select u.Notifications).ToList();
         }
 
-        public bool UpdateNotification(int userId, int topicId, TopicFormModel model)
+        public bool UpdateNotification(int changedBy, int topicId, TopicFormModel model)
```

Comment:
I think you don't got the Idea of the message in general:
You can't solve this task in a general way like this, You have to ensure which attributes was changed.
For example you create even yourself a notification that the topic was changed.
Furthermore every user should only get one notification per change, also if he's assigned twice.
Currently on Add and Delete you create no notifications. 
Only on Change but on every change you say: "You are now related to a new topic ..." but this isn't always the case.
It's also possible that the e.g. the Content was changed.